.github/workflows/aws.yml

```diff
@@ -69,14 +71,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: "quivr"
-            service: "quivr-backend"
+          - name: "quivr-api"
```

Comment:
no quivr in this file. 
replace with `sr-vt-prod-api`

Reply:
Ok, I will update this.